srcpkgs/libnma/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libnma'
+pkgname=libnma
+version=1.8.30
+revision=1
+build_style=meson
+build_helper="gir"
+hostmakedepends="gettext glib-devel gtk-doc pkg-config vala"
+makedepends="gcr-devel gtk+3-devel iso-codes
+ mobile-broadband-provider-info NetworkManager-devel"
+depends="NetworkManager iso-codes"
+short_desc="NetworkManager panel applet for GNOME - runtime library"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="GPL-2.0-only, LGPL-2.1-or-later"
+homepage="https://gitlab.gnome.org/GNOME/libnma"
+distfiles="https://gitlab.gnome.org/GNOME/libnma/-/archive/${version}/libnma-${version}.tar.gz"
+checksum=c874d83dd59468b68c79ff20e8f5a689a009c89b618e89b5939fbdc81776fbf5
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+="-Dgtk_doc=false"
+	hostmakedepends+=" libnma"
+fi
+
+post_install() {
+	if [ "$CROSS_BUILD" ]; then
+		vmkdir usr/share/gtk-doc/html 0755
+		vcopy /usr/share/gtk-doc/html/libnma usr/share/gtk-doc/html
+		vmove usr/lib/girepository-1.0
```

Comment:
why is this only being installed for cross builds? should it not just always be installed?

Reply:
That last line is a mistake and doesn't need to be there. As for the first two, the package fails to build documentation properly only on ARM architectures. Those lines install files that are only missing when built on ARM.

Reply:
yeah I was following the convo in #xbps regarding the docs. the gir line is what I was referring to.

Reply:
Yeah, that'll be removed on my next push.